code/modules/projectiles/guns/energy/pulse.dm

```diff
@@ -3,6 +3,8 @@
 	desc = "A heavy-duty, multifaceted energy rifle with three modes. Preferred by front-line combat personnel."
 	icon_state = "pulse"
 	inhand_icon_state = null
+	worn_icon_state = null
+	worn_icon = 'icons/mob/clothing/back.dmi'

```

Comment:
this is already handled by `build_worn_icon` via the slot flag, remove it.

Reply:
Reverted. Had the worn_icon set to make the other slots use a working icon. I'll add it to the list of missing textures.